ble/Gap.h

```diff
@@ -161,6 +161,9 @@ class Gap {
 
     typedef FunctionPointerWithContext<bool> RadioNotificationEventCallback_t;
 
+    typedef FunctionPointerWithContext<const Gap *> GapShutdownCallback_t;
```

Comment:
I'm not sure that Gap should be const here, the user will want to remove its own callback from the shutdown callback.